apps/accounts/automations/push_account/manager.py

```diff
@@ -21,7 +21,10 @@ def method_type(cls):
         return AutomationTypes.push_account
 
     def get_secret(self, account):
-        return account.secret
+        secret = account.secret
+        if not secret:
+            secret = super().get_secret(account)
+        return secret
 
     def gen_account_inventory(self, account, asset, h, path_dir):
         secret = self.get_secret(account)
```

Comment:
The given code has no syntax errors or irregularities. The suggested change is to improve the docstring in the `get_secret` function with better explanation of parameters and return values, which could be helpful for other developers reading this code. It also provides examples about how to use the method.

Since I am checking codes from before September 2021, there might still have been some changes made in version control systems, so it's important to update any recent versions to access these updated functionalities.